### ISSUE TITLE
Commands aren't read by the Contexts app

### ISSUE THREAD

**Issue by cazssiew** [folivoraAI/BetterTouchTool#2589]
Hi, 
I'm assuming this is an issue with the Contexts app since this bug seems to only apply to it, but I haven't been able to get much help from them so I'm wondering if you might have any insight into it.

For about a month now, my trackpad commands in BetterTouchTool don't seem to be communicated to the Contexts app. Any shortcut (cmd+tab, cmd+@, option+tab, etc.) which is bound to it functions if I type it directly with a keyboard, but bypasses the Contexts app if I trigger it through a BetterTouchTool command.

In other words, if I TYPE cmd+tab, the Contexts app shows up, but if I trigger it through a BetterTouchTool command, the native app switcher shows up. 

Any ideas as to why this might be happening ? Thanks for the awesome app !

**Comment by cazssiew** [comment]
OK, so actually, I went through the old BTT releases, and this problem seems to appear in version 2.345. I'll be sticking to version 2.344 for the time being as this trigger is extremely useful for me. Feel free to ask any questions if my description wasn't clear.

Thanks!
